Makefile.ci

```diff
@@ -0,0 +1,17 @@
+# -*- Makefile -*-
+
+# Usage in Jenkins:
+# make -f Makefile.ci package
+
+.PHONY: package
+package: obs-package-from-git
+	./$^ \
+		-P YaST:Head -p snapper \
+		-o .obsdir \
+		-c 'make -f Makefile.repo && make package'
+
+# A script that is shared with other projects and should be always refreshed
+URL=https://raw.githubusercontent.com/openSUSE/obs-package-from-git/master/obs-package-from-git
+.PHONY: always_out_of_date
+obs-package-from-git: always_out_of_date
+	wget -O $@ $(URL) && chmod +x $@
```

Comment:
no as we wget always script from master. But of course if you modify this Makefile.ci here, you have problem. Exactly same problem as now if someone create malicious stuff in configure.ac. So it is more problem of running jenkins for such kind of pull request then this pr itself.